spec/gemfury/app_spec.rb

```diff
@@ -130,14 +130,13 @@ def app_should_die(*args)
   end
 
   def stub_uploads
-    body = fixture('upload.json').read
-    stub_post("uploads", 2).to_return(:body => body)
-    stub_put("uploads/WTFBBQ123", 2).to_return(:body => body)
+    stub_post(nil, :endpoint => Gemfury.pushpoint).
```

Comment:
Leaving path argument as `nil` expects a request to root path for `Gemfury.pushpoint`. It should be `uploads`. Something's amiss here if the tests pass.